source/payment_flow/index.html.md.erb

```diff
@@ -318,30 +318,23 @@ The `state` array within the JSON lets you know the outcome of the payment:
 See the [__Making payments__](/making_payments/#making-payments) section for
 more information about how you can integrate your service with GOV.UK Pay.
 
-## Resume a payment
+### Resuming an incomplete payment
 
-If your user starts a payment but does not complete it, they can resume that
-incomplete payment. An example of this could be if they use an internet
-browser’s ‘back’ button during a payment, and then go forward by selecting
-links on your website.
+You can send your user back to GOV.UK Pay if they arrive at your `return_url` before they've finished their payment. For example, your user selected their web browser’s **Back** button while making a payment, then selected **Forward**.
 
-If your service uses the resume payment feature, you will:
+You can only resume a payment if the `status` is one of the following:
 
-* minimise the number of expired payments
+- `created`
+- `started`
+- `submitted`
 
-* reduce the number of unnecessary new payments
+Resume the payment by [calling the GOV.UK API to find the payment](/reporting/#get-information-about-a-single-payment), then redirecting your user to the new `next_url` from the API response.
```

Comment:
Might be good to add that the service will want to check the status of the payment when they get it from the API. Maybe this sentence could be put before the list of resumable statuses (so it’s something along the lines of: “You can get the payment and check its status. If its status is one of these then there will be a `next_url` you can send the user to.”).

Reply:
Good plan. I've had a go at this in https://github.com/alphagov/pay-tech-docs/pull/332/commits/7924b40b4e4d63cc7d3dd4078cc21be51d6e28b6 but I'm still not sure I'm 100% there. Let me know what you think!